src/views/VisitsView.vue

```diff
@@ -439,51 +440,58 @@ export default {
         this.canLoadMore = false;
         return;
       }
-      const eventsByDevice = result.rows;
-      if (mergeResults) {
-        this.count += result.numVisits;
-      } else {
-        this.count = result.numVisits;
-      }
       this.offset = result.queryOffset;
       this.canLoadMore = result.hasMoreVisits;
+
+
+      const summary = result.summary;
+      const filtered = result.visits.filter(
+        (visit: Visit): boolean =>
+          visit.events.length > 0 &&
+          visit.what != DefaultLabels.allLabels.bird.value &&
+          visit.what != DefaultLabels.allLabels.falsePositive.value
+      );
+      this.count += filtered.length;
       if (this.count > 0) {
```

Comment:
This kind of view logic is probably more appropriate in the actual template.

Reply:
Definately